packages/vite/src/node/cli.ts

```diff
@@ -50,6 +52,34 @@ function cleanOptions(options: GlobalCLIOptions) {
   return ret
 }
 
+function listenRestart(server: ViteDevServer) {
+  process.stdin.resume()
+  process.stdin.setEncoding('utf8')
+  //   process.stdin.setRawMode(true)
+  process.stdout.write(`
+  Available commands:
+    Press "r" to restart the dev server.
+    Press "f" to rebuild the optimized dependency cache.
+    Press "o" to open the browser.
+  `)
```

Comment:
**suggestion:** I think at least this should be executed with the same info logger as the other two logs above this

Please orientate yourself on these lines: https://github.com/vitejs/vite/blob/ab0865223f2fd8c460ee9b5108dbe67b9ad534a1/packages/vite/src/node/logger.ts#L132-L134